code/app/src/main/java/com/example/bread/utils/EmotionUtils.java

```diff
@@ -0,0 +1,26 @@
+package com.example.bread.utils; // Ensures proper package placement
+
+import java.util.HashMap;
+
+public class EmotionUtils {
+    // used a hashmap here to basically store the emotion-to-emoticon mappings
+    private static final HashMap<String, String> emotionEmoticonMap = new HashMap<>();
+
+    // Static block for the mappings when the class is first accessed
+    static {
+        emotionEmoticonMap.put("HAPPY", "😃");      // Happy
+        emotionEmoticonMap.put("SAD", "😢");        // Sad
+        emotionEmoticonMap.put("ANGRY", "😡");      // Angry
+        emotionEmoticonMap.put("ANXIOUS", "😰");    // Anxious
+        emotionEmoticonMap.put("NEUTRAL", "😐");    // Neutral
+        emotionEmoticonMap.put("CONFUSED", "😕");   // Confused
+        emotionEmoticonMap.put("FEARFUL", "😨");    // Fearful
+        emotionEmoticonMap.put("SHAMEFUL", "😞");   // Shameful
+        emotionEmoticonMap.put("SURPRISED", "😲");  // Surprised
+    }
+
+
+    public static String getEmoticon(String emotion) {
```

Comment:
Instead of referencing the emoticon using the string I would recommend using the already existing enum in `MoodEvent` class so that we keep it consistent throughout our codebase.